i18n/nl/cosmic_settings.ftl

```diff
@@ -0,0 +1,807 @@
+app = COSMIC-instellingen
+
+dbus-connection-error = Kon geen verbinding maken met DBus
+ok = Oké
+unknown = Onbekend
+
+number = { $number }
+
+## Network & Wireless
+
+connections-and-profiles = { $variant ->
+    [wired] Bekabelde verbindingen
+    [wifi] Wifi
+    [vpn] VPN
+    *[other] Onbekende verbindingen
+} en verbindingsprofielen.
+
+add-network = Netwerk toevoegen
+    .profile = Profiel toevoegen
+add-vpn = VPN toevoegen
+airplane-on = Vliegtuigmodus is ingeschakeld.
+cable-unplugged = Kabel losgekoppeld
+connect = Verbinden
+connected = Verbonden
+connecting = Verbinding maken...
+disconnect = Verbinden verbreken
+forget = Vergeten
+known-networks = Bekende netwerken
+network-and-wireless = Netwerk en wifi
+no-networks = Er zijn geen netwerken gevonden.
+no-vpn = Geen VPN-verbindingen beschikbaar.
+password = Wachtwoord
+remove = Verwijderen
+settings = Instellingen
+username = Gebruikersnaam
+visible-networks = Zichtbare netwerken
+
+auth-dialog = Authenticatie vereist
+    .vpn-description = Voer de gebruikersnaam en het wachtwoord van de VPN-dienst in.
+    .wifi-description = Voer het wachtwoord of de encryptiesleutel in. U kunt ook verbinden door op de "WPS"-knop op de router te drukken.
```

Comment:
Line 40: use formal "U" (you) instead of informal "je / jij" (you):
What is the correct way?